pkg/upstream/helm.go

```diff
@@ -268,10 +269,11 @@ func addReplicatedValues(doc *yaml.Node, replicatedChartName string, isSubchart
 
 func buildReplicatedValues(u *types.Upstream, options types.WriteOptions) (map[string]interface{}, error) {
 	replicatedValues := map[string]interface{}{
-		"replicatedID": options.KotsadmID,
-		"appID":        options.AppID,
-		"userAgent":    buildversion.GetUserAgent(),
-		"isAirgap":     options.IsAirgap,
+		"replicatedID":              options.KotsadmID,
+		"appID":                     options.AppID,
+		"userAgent":                 buildversion.GetUserAgent(),
+		"isAirgap":                  options.IsAirgap,
+		"additionalMetricsEndpoint": fmt.Sprintf("%s/api/v1/app/metrics", fmt.Sprintf("http://kotsadm.%s.svc.cluster.local:3000", util.PodNamespace)),
```

Comment:
Anything else we should do here to support flows that don't have a cluster context? Looking here for example: https://github.com/replicatedhq/kots/blob/main/pkg/supportbundle/spec.go#L359-L368

Reply:
The `additionalMetricsEndpoint` will only be used within the SDK's API, and currently, that can only run as a service inside the cluster. I don't think there are use cases without a cluster context when things are managed by KOTS.